go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	github.com/gorilla/websocket v1.4.2
 	github.com/gosimple/slug v1.9.0
 	github.com/grafana/grafana-aws-sdk v0.7.0
-	github.com/grafana/grafana-plugin-sdk-go v0.112.0
+	github.com/grafana/grafana-plugin-sdk-go v0.112.1-0.20210907204529-6d0be6b7893a
```

Comment:
@wbrowne -- can you review this PR assuming we link it to a new release?  I think it is ready to go.  Thanks for your help!!!